[CUBRIDMAN-317] plandump 설명 추가 및 수정 (#676)
CUBRID/cubrid-manual: en/admin/admin_utils.rst

        cubrid plandump -s 'da9cab5d 597f2fe4 48aa8ae8 66b87d81 49fdd7ca' demodb

> If no option is used, it checks the query plans stored in the cache. ::

Reviewer: ```suggestion
By default, it examines the query plans stored in the plan cache if no options are specified. ::
```

Author: @mhoh 영문을 재 수정하여 반영하였습니다.

    2.   XASL_ID Generation : Based on the rewritten query, a XASL_ID is generated. This XASL_ID serves as a unique identifier for the query and is used to check whether the corresponding execution plan (XASL) is already stored in the server’s Plan Cache.
    3.   Plan Cache Lookup : Using the generated XASL_ID, the CAS queries the server to determine if a matching XASL (Execution Plan) already exists in the Plan Cache.
        *   If the XASL is found: The server uses the existing plan to execute the query.
        *   If the XASL is not found: The CAS generates a new Tree(Execution Plan) and sends it to the server, which may then decide whether or not to store it in the Plan Cache.

.. note::

    The Plan Cache improves performance by reusing XASL(execution plan).

The following describes the detailed fields included in the output of plandump:

*   XASL_ID : The SQL hash text is converted into a SHA1 hash string, which is then used as the key for the plan cache.

> 	*	The time_stored field within the XASL_ID indicates the time when the plan was stored in the cache.(unix timestamp)

Reviewer 2: ```suggestion
	*	The time_stored field within the XASL_ID indicates the time when the plan was stored in the cache (Unix timestamp).
```

Author: @junsklee  반영하였습니다. 리뷰 감사합니다

*   fix_count : Number of concurrently used threads
*   cache flags : Status information of the cache entry

    *   8000000 : mark_deleted
    *   4000000 : to be recompiled
    *   2000000 : was recompiled
    *   0800000 : clean up
    *   0400000 : recomiled requested

> *   reference count : The number of times the entry has been referenced.(unix timestamp)

Reviewer 2: ```suggestion
*   reference count : The number of times the entry has been referenced (Unix timestamp).
```

Author: 반영하였습니다. 리뷰 감사합니다

*   time second last used : UNIX timestamp of the last usage of this plan. Convert to readable format using: date -d @<timestamp>
*   clone count : Number of clone instances created from this plan.

> *   SQL_ID : The SQL ID is generated by converting the original SQL statement into a 32-character hexadecimal string using the MD5 hash function. The last 13 hexadecimal digits of this hash value are used as the SQL ID.

Reviewer 2: ```suggestion
*   SQL_ID : Generated by converting the original SQL statement into a 32-character hexadecimal string using the MD5 hash function. The last 13 hexadecimal digits of this hash value are used as the SQL ID.
```

Author: 반영하였습니다. 리뷰 감사합니다

*   sql user text : Original SQL statement string

> *   sql hash text : This is the rewritten SQL text

Reviewer 2: ```suggestion
*   sql hash text : The rewritten SQL text
```

Author: _반영하였습니다. 리뷰 감사합니다_

*   sql plan text : Execution Plan
*   OID_LIST : Object ID list referenced by the execution plan

.. note::

    *   This timestamp can be converted and verified using the following command

        *   $>date -d @1750657813 
        *   $> date -d @1750657813.784232

>     *   Additional explanation for clone count

Reviewer 2: ```suggestion
    *   Clone count (detailed explanation)
```

Author: 반영하였습니다. 리뷰 감사합니다
